library/pod-security-policy/volumes/template.yaml

```diff
@@ -15,6 +15,7 @@ spec:
           type: object
           properties:
             volumes:
+              description: "`volumes` is an array of volume types, recorded as strings.  All volume types can be enabled using `*`."
```

Comment:
No need to list the type ("recorded as strings"), it's already in the schema metadata.

Reply:
Removed.